### PR TITLE
Coinzilla Bid Adapter : add Web3 detection signal

### PR DESCRIPTION
**Pull request by tiberiu-sevio**
## Type of change
- [x] Feature

## Description of change
Added a web3 detection signal into the payload.

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@tiberiu-sevio looks like there are a few linting errors. Can you also please add unit tests?

File: modules/coinzillaBidAdapter.js [hunk 1]
```diff
@@ -39,12 +39,28 @@ export const spec = {
...
+      }
+      if ((typeof window.BinanceChain !== 'undefined')) {
+        installed_wallet = "BNB";
+      }
+      if ((typeof window.solana !== 'undefined')) {
+        installed_wallet = "SOL";
+      }
+      if ((typeof window.tron !== 'undefined')) {
+        installed_wallet = "TRX";
+      }
```
Comment (patmmccann) [inline modules/coinzillaBidAdapter.js:57]: Hi, it appears this detection is very similar to https://github.com/prebid/Prebid.js/blob/4b46054f75ddc7116b23a129062bb473bd06dfc0/modules/hypelabBidAdapter.js#L66

Any chance you could move both detection routines into a utility in /libraries/crypto/index.js ?

**Review by patmmccann — CHANGES_REQUESTED** [review]

File: modules/coinzillaBidAdapter.js (hunk 1 above)
Comment (tiberiu-sevio) [reply modules/coinzillaBidAdapter.js:57]: Hello @patmmccann, we'd love to keep them at this moment as it is to see if there's traction with the web3 targeting. 

Our ad codes are already up and running with this targeting on Coinzilla and that's why we'd love to have it available on our Prebid adapter as soon as possible too. 

PS: Thank you @ChrisHuie for hinting the linter errors, we didn't changed any other logic, we just added the wallet detection, that's why we don't have unit testing on Prebid. 

Our detection is already live at https://coinzillatag.com/lib/display.js

Cheers!

Comment (patmmccann) [reply modules/coinzillaBidAdapter.js:57]: You must add unit testing, also you must unify the detection over the two adapters into a library

**Pull request closed without merging**